test/tsconfig.json

```diff
@@ -8,20 +8,19 @@
         "outDir": "../build"
     },
     "atom": {
-        "rewriteTsconfig": false
+        "rewriteTsconfig": true
     },
     "filesGlob": [
         "../typings/tsd.d.ts",
-        "./typings/**/*.d.ts",
+        "./typings/tsd.d.ts",
         "../src/**/*.ts",
-        "/*.ts",
+        "./*.ts",
         "./rules/*.ts",
         "./formatters/*.ts"
     ],
     "files": [
         "../typings/tsd.d.ts",
-        "typings/chai/chai.d.ts",
-        "typings/mocha/mocha.d.ts",
+        "./typings/tsd.d.ts",
```

Comment:
I kind of prefer not using tsd.d.ts and instead just having typings referenced here. It feels weird to have one file that contains `///` references while no other files do. But tsd.d.ts is also sort of a standard, so either way is fine